libdlib/package.xml

```diff
@@ -7,7 +7,7 @@
 	</description>
 	<author email="florian.jordan@ipa.fraunhofer.de">Florian Jordan</author>
 	<maintainer email="richard.bormann@ipa.fraunhofer.de">Richard Bormann</maintainer>
-	<license>LGPL</license>
+	<license>Boost Software License</license>
```

Comment:
Is this the correct license?
I thought convention is to normally keep the wrapper package `LGPL`...as we keep the actually library as is...